src/pages/season1recap.tsx

```diff
@@ -0,0 +1,163 @@
+import { GetStaticProps } from 'next'
+import { PageMetaProps } from '../components/global/Head'
+import styled from 'styled-components'
+
+declare global {
+  interface Window {
+    attachEvent: any
+  }
+}
+
+const Container = styled.div`
+  height: 100vh;
+  max-height: 100vh;
+  overflow-x: scroll;
+  overflow-y: hidden;
+`
+
+const Intro = styled.img`
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+`
+
+const Scroll = styled.div`
+  position: absolute;
+  top: 0;
+  left: 100vw;
+  min-width: 100vw;
+  height: 100vh;
```

Comment:
100vh has issues with the horizontal scroll bar: a slight overlap causes a vertical scroll bar to show. Need to switch to 100% height, 98vh or some alternative solution

Reply:
Difficult for me to test because I'm on a Mac (scroll bars are transparent).
Can you test https://github.com/BanklessDAO/season2.bankless.community/pull/1 preview link and check if this is still a problem?
Thanks!